src/src.pro

```diff
@@ -100,14 +100,16 @@ unix:!macx {
 } else:win32: {
     LIBS += -L"C:\\mudlet5_package" \
         -L"C:\\mingw32\\lib" \
-        -llua51 \
+        -llua \
         -lpcre \
-        -lhunspell \
-        -llibzip \
-        -lzlib \
-        -llibzip \
+        -llibhunspell-1.3 \
```

Comment:
Are you SURE it has to be __lib__hunspell - on *nix platforms the lib prefix is a given IIRC!  Also does it HAVE to be version __1.3__ and not anything earlier or later (unlike the lua dependency which MUST be 5.1)?